pkg/reconciler/common/releases.go

```diff
@@ -230,27 +253,17 @@ func targetManifestPath(version string, instance v1alpha1.KComponent) string {
 			return ""
 		}
 	}
-
-	// Append the spec.additionalManifests
-	addManifests := instance.GetSpec().GetAdditionalManifests()
-	urls = make([]string, 0, len(addManifests))
-	urls = append(urls, manifestPath)
-	for _, manifest := range addManifests {
-		url := strings.ReplaceAll(manifest.Url, VersionVariable, version)
-		urls = append(urls, url)
-	}
-
-	return strings.Join(urls, COMMA)
+	return manifestPath
 }
 
 func targetManifestPathArray(instance v1alpha1.KComponent) []string {
-	if len(instance.GetSpec().GetManifests()) > 0 || len(instance.GetSpec().GetAdditionalManifests()) > 0 {
-		// If either spec.manifests or spec.additionalManifests is not empty, we leverage status.manifests
-		// to save the complete manifest path.
-		return strings.Split(targetManifestPath(TargetVersion(instance), instance), COMMA)
+	pathArrayPath := targetManifestPath(instance)
```

Comment:
`pathArrayPath` sounds strange. Also `pathArrayPath` does not have array but comma separated strings, correct?

Reply:
I changed it into target path.